site/router-topology/router-encapsulation.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Weave Router Encapsulation
+layout: default
+---
+
+
+When the Weave router forwards packets, the encapsulation looks
+something like this:
+
+    +-----------------------------------+
```

Comment:
this diagram looks like site/images/weave-frame-encapsulation-178x300.png ; was it supposed to be replaced?